packages/headless/src/controllers/tab/headless-tab.ts

```diff
@@ -26,15 +41,26 @@ const initialStateSchema = new Schema({
 });
 
 export interface TabProps {
+  /**
+   * The options for the `Tab` controller.
+   */
   options: TabOptions;
+  /**
+   * The initial state that should be applied to this `Tab` controller.
+   */
   initialState?: TabInitialState;
 }
 
-export type TabInitialState = SchemaValues<typeof initialStateSchema>;
-
 export type Tab = ReturnType<typeof buildTab>;
+/**
+ * A scoped and simplified part of the headless state that is relevant to the `Tab` controller.
+ */
 export type TabState = Tab['state'];
 
+/**
+ * The `Tab` headless controller offers a high-level interface that allows the end user to select a specific search interface.
```

Comment:
I find "high-level interface" a confusing phrase and I do not think we need to mention it.

Also, would it be better to mention that the tab allows selecting a subset of results instead of a search interface?
e.g.
```
The `Tab` headless controller allows the end user to view a subset of results.
```